### ISSUE TITLE
ReCaptcha multi-instance

### ISSUE THREAD

**Issue by esprit-dev** [Lansoweb/LosReCaptcha#7]
Hi,

I'm using your ReCaptcha module and it works pretty good :)
But i had to find a workaround for keeping out conflits when using multiple form with captcha in the same page, as clones, which have no ID setted on captcha element. In this case, name of the element is taken into account, and there is conflicts betweens same forms.

Could you give me your feedback ? can you upgrade your module for multi-instance usage ?
I can give you my patch file if necessary, that can help you to understand my request.

Thx

**Comment by Lansoweb** [comment]
@esprit-dev Hi! Sorry for the delay! Yes, paste your solution, also, i'll check with the official zend recaptcha (they releases a new version with v2 support this month).
And thanks for your support!

**Comment by Marc-pi** [comment]
> i'll check with the official zend recaptcha (they releases a new version with v2 support this month).

yes, invisible recaptcha is needed too :-)

**Comment by Lansoweb** [comment]
@Marc-pi True, but it's not released yet :-) For others, that's the [invislbe recaptcha](https://www.google.com/recaptcha/intro/comingsoon/invisible.html)

**Comment by Lansoweb** [comment]
Multiple recaptchas is possible through the "explicit" mode as in [documentation](https://developers.google.com/recaptcha/docs/display#explicit_render). Will work on that.

**Comment by Marc-pi** [comment]
yep, @esprit-dev work is there https://github.com/Marc-pi/pi/tree/develop/lib/vendor/LosReCaptcha
i think we used this explicit method too, see https://github.com/Marc-pi/pi/commit/f5e6dba6241b91256d58e9c5123e2536ce412dd4

**Comment by Lansoweb** [comment]
@Marc-pi Great! Thanks, will work on an adaptation for this module soon. Nice work.

**Comment by Lansoweb** [comment]
@Marc-pi Hi! The invisible captcha is GA. Will add it to the lib soon.

**Comment by Marc-pi** [comment]
yes, i've noticed also, hope to see the lib improvements soon (multi-instance and invisible)

**Comment by Marc-pi** [comment]
we fixed one W3C error here
https://github.com/pi-engine/pi/commit/701d4206d58a104ab11d3bb3142beec70e0a814d

**Comment by Marc-pi** [comment]
@Lansoweb did you find time to add the new invisible ?

**Comment by Marc-pi** [comment]
hello,
any news on integrating invisible recaptcha ?

**Comment by Lansoweb** [comment]
Released [2.1.0](https://github.com/Lansoweb/LosReCaptcha/releases/tag/2.1.0) with invisible recaptcha. You can give it a try.

**Comment by Marc-pi** [comment]
good, we will try.
for min Zend version, what did you set (we do not run 3x for now, we are on 2.4.10) ?

**Comment by Lansoweb** [comment]
I set only zend-form and zend-captcha:
```json
"zendframework/zend-form": "^2.11",
"zendframework/zend-captcha": "^2.7"
```
So it **should** work with either zf version.

**Comment by Marc-pi** [comment]
ok, thx
did you keep multi-instance (see above) ??
